pkgs/applications/science/chemistry/pymol/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchurl, python27Packages, glew, freeglut, libpng, libxml2, tk, freetype }:
+
+let
+  version = "1.8.4.0";
+in
+python27Packages.buildPythonPackage {
+  name = "pymol-${version}";
+  src = fetchurl {
+    url = "mirror://sourceforge/project/pymol/pymol/1.8/pymol-v1.8.4.0.tar.bz2";
+    sha256 = "0yfj8g5yic9zz6f0bw2n8h6ifvgsn8qvhq84alixsi28wzppn55n";
+  };
+
+  buildInputs = [ python27Packages.numpy glew freeglut libpng libxml2.dev tk freetype ];
```

Comment:
Adding `libxml2` to `buildInputs` should automatically pull `libxml2.dev` as well.